Testing/functional/tests/cypress/integration/ui/pages/home.js

```diff
@@ -18,6 +19,11 @@ describe("Authenticated User - Home Page", () => {
 
     it("Home - Federal Card button enabled", () => {
         cy.enableModules(["FederalCardButton"]);
+        cy.configureSettings({
+            homepage: {
+                showFederalProofOfVaccination: true,
+            },
+        });
```

Comment:
The enableModules call should be rolled into configureSettings.

Reply:
Fixed in Fixed in AB#14995